core/src/main/java/com/scalar/db/storage/dynamo/Dynamo.java

```diff
@@ -66,10 +66,13 @@ public Dynamo(DatabaseConfig databaseConfig) {
             new DynamoAdmin(client, config), databaseConfig.getMetadataCacheExpirationTimeSecs());
     operationChecker = new DynamoOperationChecker(metadataManager);
 
-    selectStatementHandler = new SelectStatementHandler(client, metadataManager);
-    putStatementHandler = new PutStatementHandler(client, metadataManager);
-    deleteStatementHandler = new DeleteStatementHandler(client, metadataManager);
-    batchHandler = new BatchHandler(client, metadataManager);
+    selectStatementHandler =
```

Comment:
Ideally, I would have preferred to add the logic to append the namespace prefix in the `Dynamo` class, but since each handler class performs an `admin.getTableMetadata()` call which expects an unprefixed namespace, I decided to put the namespace prefix appending logic in each handler classes.